Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:14.04
+
+MAINTAINER Kyle Dove <dovek@umich.edu>
+
+# Fix sh
```

Comment:
I'd prefer a better comment - you're blowing away the default sh and linking it to bash - why not use `chsh -s /bin/bash`